test/integration-tests/integration-tests.js

```diff
@@ -0,0 +1,129 @@
+const chai = require('chai');
+const base = require('./base');
+const utilities = require('./test-utilities');
+
+// eslint-disable-next-line func-names
+describe('Integration Tests', function () {
+  this.timeout(15 * 60 * 1000);
+
+  beforeEach(async () => {
+    this.logsReceiverName = `${base.PLUGIN_IDENTIFIER}-${base.RANDOM_STRING}-logs-receiver`;
+    this.logsReceiverArn = await utilities.createDummyFunction(this.logsReceiverName);
+    await utilities.ensureFunctionCreated(this.logsReceiverName);
+    await utilities.pingFunction(this.logsReceiverName);
+    process.env.LOGS_RECEIVER_ARN = this.logsReceiverArn;
+  });
+
+  afterEach(async () => {
+    await utilities.deleteFunction(this.logsReceiverName);
```

Comment:
Is there a way to create the function once at the start of the whole test suite instead of at the start of each test case?

Reply:
Not really. Because the plugin assigned a policy to a function. Technically it's possible but I think it's not fair